File: ch6/sigdemo2.c
#include<stdio.h>
#include<signal.h>
#include<unistd.h>

int main(int argc, char const *argv[])
{
    signal(SIGINT,SIG_IGN);
    printf("you can't stop me!\n");
    while (1)
    {
        sleep(1);
        printf("jiejiejieje\n");
    }
    return 0;
}

File: ch6/ch6_src/src/bin/sigdemo2.rs
use std::{thread::sleep, time::Duration};

use libc::{signal, SIGINT, SIG_IGN};

fn main() {
    unsafe {
        signal(SIGINT, SIG_IGN);
    }
    println!("you can't stop me!");
    loop {
        sleep(Duration::from_secs(1));
        println!("jiejiejieje");
    }
}
